python/rascaline/examples/le-basis.py

```diff
@@ -0,0 +1,197 @@
+"""
+.. _example-le-basis:
+
+Laplacian eigenstate basis
+==========================
+
+.. start-body
+"""
+
+# %%
+#
+# This script illustrates how to generate a spherical expansion using
+# the Laplacian eigenstate basis (https://doi.org/10.1063/5.0124363),
+# both using truncation with l_max, n_max hyper-parameters and with an
+# eigenvalue threshold.
```

Comment:
You can move this into the comment section above. Makes it maybe even more readable.

```suggestion
"""
This script illustrates how to generate a spherical expansion using
the Laplacian eigenstate basis (https://doi.org/10.1063/5.0124363),
both using truncation with l_max, n_max hyper-parameters and with an
eigenvalue threshold.
"""

# %%
```